MoreLinq.Test/WindowedTest.cs

```diff
@@ -24,22 +24,26 @@ public void TestSlidingWindowIsLazy()
         /// Verify that invoking SlidingWindow on a <c>null</c> sequence results in an exception
         /// </summary>
         [Test]
-        [ExpectedException(typeof(ArgumentNullException))]
         public void TestSlidingWindowNullSequenceException()
         {
-            const IEnumerable<int> sequence = null;
-            sequence.Windowed(10);
+            Assert.Throws<ArgumentNullException>(() =>
+            {
+                const IEnumerable<int> sequence = null;
+                sequence.Windowed(10);
+            });
         }
 
         /// <summary>
         /// Verify that a negative window size results in an exception
         /// </summary>
         [Test]
-        [ExpectedException(typeof(ArgumentOutOfRangeException))]
         public void TestSlidingWindowNegativeWindowSizeException()
         {
-            var sequence = Enumerable.Repeat(1, 10);
-            sequence.Windowed(-5);
+            Assert.Throws<ArgumentOutOfRangeException>(() =>
+            {
+                var sequence = Enumerable.Repeat(1, 10);
+                sequence.Windowed(-5);
```

Comment:
Only this line should be inside the lambda to Assert.Throws as that's what will cause and being tested for an exception.